lib/osf-components/addon/components/brand-color-inputs/template.hbs

```diff
@@ -0,0 +1,5 @@
+{{!-- template-lint-disable no-bare-strings  --}}
+<label>Brand Primary Color</label>
+<Input @value={{this.primaryColor}} />
+{{!-- template-lint-disable no-bare-strings  --}}
+<Button {{on 'click' this.setPrimaryColor}}>Set Primary Color</Button>
```

Comment:
I think you could add this string into `translations/en-us.yml` in case we want to add more properties into this component eventually, but not  a deal-breaker for me